packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -245,6 +246,9 @@ export const cicdContent = (
   <EntitySwitch>
     <EntitySwitch.Case if={isJenkinsAvailable}>
       <EntityJenkinsContent />
+      <hr />
+      <hr />
+      <EntityJobRunsTable />
```

Comment:
I think this integrations needs a bit more consideration. The `EntityJenkinsContent` is a full-screen component that should not be mixed with additional content. Do you think it makes sense to integrate this table directly into the [main content as a separate subroute](https://github.com/backstage/backstage/blob/0870577bd4ac6922ff999a0207c2aed71512d499/plugins/jenkins/src/components/Router.tsx#L42)?

Reply:
I have now made a new route for this clicking on action history icons takes us to the new route showing this component which all the build runs
This look awesome now. 

:heart_eyes: :heart_eyes: